src/components/Dropdown/Dropdown.js

```diff
@@ -44,15 +45,6 @@ const ItemWrapper = Item.extend`
   &:hover {
     background-color: #eee;
   }
-
-  &:first-child {
```

Comment:
Why do these need to be changed?

Reply:
Because of https://github.com/weaveworks/ui-components/pull/108/files#diff-dae838509391e35fda596d1211449cdbR28, the background of a selected item no longer touches the rounded corner